modules/Microsoft.SignalRService/webPubSub/.test/min.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
```

Comment:
Since location is not a required parameter I'd suggest to remove it

Reply:
done